client/my-sites/plugins/main.jsx

```diff
@@ -426,7 +425,7 @@ const PluginsMain = React.createClass( {
 export default connect(
 	state => {
 		return {
-			wporgPlugins: state.plugins.wporg
+			plugins: state.plugins.wporg
```

Comment:
I think we should keep the wporgPlugins name  (or choose any other one more descriptive than just `plugins`).  `plugins` can be confusing because we have two different concepts that would fall under that category. On one side, there's the user plugins, the list of plugins a user have installed on their sites. On the other side, we have wporgPlugins, which is the information about those plugins we retrieve from .org API. The first list only contain information about the plugins state (if it's activated or not, the site where it is installed, etc) and the second one deals with generic plugin information (description, images, author, etc) but not with single installs state.

We can't merge those lists into a single one, because we would be storing repeated data (if a user has Akismet, for example, installed in 30 sites, there will be 30 instances of "akismet" in the user's plugins lists), and also there can be plugins in the wporgPlugins list that are not in the user plugins list (when someone use the plugins' browser, we are retrieving those non-installed plugins info from .org and storing it in wporgPlugins too).

So ... `plugins` can be a little ambiguous name here :)